tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -109,6 +109,10 @@ class DuckDBPyConnection:
     ) -> DuckDBPyRelation: ...
     def from_df(self, df: pandas.DataFrame = ...) -> DuckDBPyRelation: ...
     @overload
+    def read_parquet(self, file_glob: str, binary_as_string: bool = ..., *, file_row_number: bool = ..., filename: bool = ..., hive_partitioning: bool = ..., union_by_name: bool = ...) -> DuckDBPyRelation: ...
+    @overload
+    def read_parquet(self, file_globs: list[str], binary_as_string: bool = ..., *, file_row_number: bool = ..., filename: bool = ..., hive_partitioning: bool = ..., union_by_name: bool = ...) -> DuckDBPyRelation: ...
```

Comment:
Might need to double check this syntax, I think `list[str]` might only be supported from 3.10? Instead of `List[str]` (yes it's just the case that's different)

Reply:
Love python 🙃 
Thanks for the heads-up, I'll fix it :P